recipe/meta.yaml

```diff
@@ -30,7 +30,7 @@ requirements:
     - jedi >=0.10
     - pexpect  # [unix]
     - pickleshare
-    - prompt-toolkit >=2.0.0,<4,!=3.0.0,!=3.0.1
+    - prompt-toolkit !=3.0.0,!=3.0.1,<3.1.0,>=2.0.0
```

Comment:
We are correctly `prompt-toolkit choosing 3.0.4` and only that one now :smile: 

I'll check the test failures tomorrow and wait for an upstream update on the version that we should pin here.